abs.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  melody_org: flask-update
```

Comment:
please remove this channels stuff from commit

Reply:
The abs.yaml has been removed